src/components/Picker/Picker.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import { Text } from '../Text';
+import { PICKER_DATA_HOOKS, PICKER_DATA_KEYS } from './dataHooks';
+import { ReactComponent as ChevronLeft } from '../../assets/icons/ChevronLeft.svg';
+import { ReactComponent as ChevronRight } from '../../assets/icons/ChevronRight.svg';
+import { IconButton } from '../IconButton';
+import styles from './Picker.st.css';
+
+export interface PickerProps {
```

Comment:
I think we'll also need to add:
```
disablePrev?: boolean;
disableNext?: boolean;
```

Reply:
or
```
disable?: 'prev' | 'next' | 'both';
```

Reply:
good idea ! what do you think about calling them prevDisabled and nextDisabled ? because they are booleans so adjectives would fit better then verbs